chat/src/Components/UsersList.svelte

```diff
@@ -6,99 +6,114 @@
+                reduced.set(group, usersList);
             }
             return reduced;
         }, new Map<string, User[]>());
 
     $: roomSorted = [...usersByMaps.keys()].sort((a, b) =>
         $me && $me.roomName === a ? -1 : $me && $me?.roomName === b ? 1 : a.localeCompare(b)
     );
-
-    $: usersLenght = usersList.filter((user: User) => {
-        if (!searchValue || searchValue === "") return true;
-        return user.name.toLocaleLowerCase().includes(searchValue);
-    }).length;
 </script>
 
-<div id="users" class="users tw-border-b tw-border-solid tw-border-0 tw-border-transparent tw-border-b-light-purple">
-    <div class="tw-px-4 tw-py-1 tw-flex tw-items-center">
-        {#if !$loadingSubscribersStore}
-            <span
-                class="tw-bg-light-blue tw-text-dark-purple tw-w-5 tw-h-5 tw-mr-3 tw-text-sm tw-font-semibold tw-flex tw-items-center tw-justify-center tw-rounded"
-            >
-                {usersList.filter((user) => user.active).length}
-            </span>
-        {/if}
-        <p class="tw-text-light-blue tw-mb-0 tw-text-sm tw-flex-auto">
-            {$LL.users()}
-        </p>
-        <button class="tw-text-lighter-purple" on:click={() => showUsersStore.set(!$showUsersStore)}>
-            <ChevronUpIcon class={`tw-transform tw-transition ${$showUsersStore ? "" : "tw-rotate-180"}`} />
+{#if [...$presenceStore.values()].filter((user) => !user.isMe && user.active).length === 0}
+    <div
+        class="tw-px-5 tw-py-4 tw-border-b tw-border-solid tw-border-0 tw-border-transparent tw-border-b-light-purple tw-text-sm tw-text-center"
+    >
+        <p>{$LL.roomEmpty()}</p>
+        <button type="button" class="light tw-m-auto tw-cursor-pointer tw-px-3" on:click={showInviteMenu}>
+            {$LL.invite()}
         </button>
     </div>
-    {#if $showUsersStore}
-        <div transition:fly={{ y: -30, duration: 100 }}>
-            {#if $loadingSubscribersStore}
-                <Loader text={$LL.loadingUsers()} height="tw-h-40" />
-            {:else}
-                {#each roomSorted as room}
-                    {#each usersByMaps.get(room) ?? [] as user}
-                        <ChatUser {mucRoom} {user} {searchValue} />
-                    {/each}
-                {/each}
-                {#if usersList.filter((user) => !user.isMe).length === 0}
-                    <div
-                        class="tw-mt-2 tw-px-5 tw-py-4 tw-border-t tw-border-solid tw-border-0 tw-border-transparent tw-border-t-light-purple"
+{/if}
+{#if $loadingSubscribersStore}
+    <Loader text={$LL.loadingUsers()} height="tw-h-40" />
+{:else}
+    {#each roomSorted as room}
+        <div class="users tw-border-b tw-border-solid tw-border-0 tw-border-transparent tw-border-b-light-purple">
+            <div class="tw-px-4 tw-py-1 tw-flex tw-items-center">
+                {#if !$loadingSubscribersStore}
+                    <span
+                        class="{room !== 'ZZZZ-Disconnected'
```

Comment:
What is the `ZZZZ-Disconnected`

Reply:
It's the name of the room of the users that are disconnected, just to be sure that this "room" will be at the end of the array when doing a sort, this is why the "ZZZZ-"

Reply:
There no possibility to force the sort, because if there are `ZZZZZ` name for one forum or anything else, it will be after disconnected room